### PR TITLE
Change machine learning note to the new procedure

### PR DESCRIPTION
**Pull request by sciencewhiz**

### CODE REVIEW

**Comment by sciencewhiz** [comment]
Supercedes #546

File: source/docs/software/examples-tutorials/machine-learning/index.rst [hunk 1]
```diff
@@ -1,7 +1,16 @@
 Machine Learning
 ================
 
-.. Warning:: We are aware that you will get an exception while trying to train your model with the Jupyter notebook. The WPILib team and Amazon are working hard on a solution and hope to have something in the next few days. Until we do, you will not be able to train your models. We will post the status here as it changes, so please check back often. We are very sorry that this issue occurred and hope to get it resolved as quickly as possible.
+.. Warning::
```
Comment (Daltz333) [inline source/docs/software/examples-tutorials/machine-learning/index.rst]:
```suggestion
.. warning:: 
```

File: source/docs/software/examples-tutorials/machine-learning/index.rst [hunk 2]
```diff
@@ -1,7 +1,16 @@
 Machine Learning
 ================
 
-.. Warning:: We are aware that you will get an exception while trying to train your model with the Jupyter notebook. The WPILib team and Amazon are working hard on a solution and hope to have something in the next few days. Until we do, you will not be able to train your models. We will post the status here as it changes, so please check back often. We are very sorry that this issue occurred and hope to get it resolved as quickly as possible.
+.. Warning:: 
+    We apologize for delay in getting the procedures to train a network to you.  AWS Educate ran into a problem with the ability to run Sagemaker on their systems at the same time build season was starting.  We have come up with a new solution below to get you going.
```
Comment (Daltz333) [inline source/docs/software/examples-tutorials/machine-learning/index.rst]:
```suggestion
   We apologize for delay in getting the procedures to train a network to you.  AWS Educate ran into a problem with the ability to run Sagemaker on their systems at the same time build season was starting.  We have come up with a new solution below to get you going.
```

File: source/docs/software/examples-tutorials/machine-learning/index.rst [hunk 3]
```diff
@@ -1,7 +1,16 @@
...
+
+    A mentor from the team will need to email us the information below, we will setup an account for them and email instructions on how to setup accounts for the students. Then continue following the instructions in this documentation.
```
Comment (Daltz333) [inline source/docs/software/examples-tutorials/machine-learning/index.rst]:
```suggestion
   A mentor from the team will need to email us the information below, we will setup an account for them and email instructions on how to setup accounts for the students. Then continue following the instructions in this documentation.
```

File: source/docs/software/examples-tutorials/machine-learning/index.rst [hunk 4]
```diff
@@ -1,7 +1,16 @@
...
+
+    Please email the following information to ML @ WPILib.org:
```
Comment (Daltz333) [inline source/docs/software/examples-tutorials/machine-learning/index.rst]:
```suggestion
   Please email the following information to ML @ WPILib.org:
```

File: source/docs/software/examples-tutorials/machine-learning/index.rst [hunk 5]
```diff
@@ -1,7 +1,16 @@
...
+
+    * Team number
```
Comment (Daltz333) [inline source/docs/software/examples-tutorials/machine-learning/index.rst]:
```suggestion
   * Team number
```

File: source/docs/software/examples-tutorials/machine-learning/index.rst [hunk 6]
```diff
@@ -1,7 +1,16 @@
...
+    * Mentors name
```
Comment (Daltz333) [inline source/docs/software/examples-tutorials/machine-learning/index.rst]:
```suggestion
   * Mentors name
```

File: source/docs/software/examples-tutorials/machine-learning/index.rst [hunk 7]
```diff
@@ -1,7 +1,16 @@
...
+    * Mentors email that has not been used for an AWS account before
```
Comment (Daltz333) [inline source/docs/software/examples-tutorials/machine-learning/index.rst]:
```suggestion
   * Mentors email that has not been used for an AWS account before
```

**Comment by AustinShalit** [comment]
This is only the start of the changes. We need to also remove all of the references to educate and add in the new procedure for adding users.

**Comment by sciencewhiz** [comment]
Ok, i was trying to help, but it seems like @GrantPerkins needs to learn rst

**Comment by GrantPerkins** [comment]
Yes, I also need to find the time to make those changes.

**Pull request closed without merging**